certif/app/components/add-student-list.js

```diff
@@ -14,26 +14,19 @@ export default class AddStudentList extends Component {
 
   @tracked selectedDivisions = this.args.selectedDivisions;
 
-  get headerCheckboxStatus() {
-    return this.hasCheckedEverything ? 'checked' : this.hasCheckedSomething ? 'partial' : 'unchecked';
+  get hasCheckState() {
+    return this._hasCheckedSomething() || this._hasCheckedEverything();
   }
```

Comment:
On peut simplifier (si `_hasCheckedEverything`, `_hasCheckedSomething` est vrai)
```suggestion
  get hasCheckState() {
    return this._hasCheckedSomething();
  }
```